provider/resources.go

```diff
@@ -5507,73 +5507,57 @@ compatibility shim in favor of the new "name" field.`)
 		},
 	}
 
-	for k, v := range pluginFrameworkResoures {
-		if _, conflict := prov.Resources[k]; conflict {
-			panic(fmt.Sprintf("Resoruce already defined: %s", k))
-		}
-		prov.Resources[k] = v
-	}
-
-	prov.MustComputeTokens(tks.MappedModules("aws_", "", moduleMap,
-		func(mod, name string) (string, error) {
-			return awsResource(mod, name).String(), nil
-		}))
-
 	prov.P.ResourcesMap().Range(func(key string, res shim.Resource) bool {
+		// only process sdkv2 resources
```

Comment:
It would be worth it, IMO, to add a bit more of the context from the PR description right here as a comment.I'm happy to have it be a followup to avoid wrestling with these massive PRs for small non-code changes.

Reply:
Good call, I'll add the comment.